ankh/main.go

```diff
@@ -431,8 +431,13 @@ func execute(ctx *ankh.ExecutionContext) {
 		executeContext(ctx, &rootAnkhFile)
 	}
 
-	if ctx.SlackChannel != "" {
-		if err := slack.PingSlackChannel(ctx); err != nil {
+	if len(ctx.AnkhConfig.Slack.Channels) > 0 {
```

Comment:
This change requires the channel to be set in the ankh config? This doesn't allow the use-case where a different slack channel is used for different application deploys.

Reply:
Updated.